h2o-r/h2o-package/R/models.R

```diff
@@ -3064,5 +3064,73 @@ h2o.deepfeatures <- function(object, data, layer) {
   h2o.getFrame(dest_key)
 }
 
+#'
+#' The H2OTree class.
+#'
+#' This class represents a model of a Tree built by one of H2O's algorithms (GBM, Random Forest).
+#' @slot left_children An \code{integer} vector with left child nodes of tree's nodes
+#' @slot right_children An \code{integer} vector with right child nodes of tree's nodes
+#' @slot descriptions A \code{character} vector with descriptions for each node to be found in the tree. Contains split threshold if the split is based on numerical column.
+#'                    For cactegorical splits, it contains list of categorical levels for transition from the parent node.
+#' @slot model_name A \code{character} string with the name of the model this tree has been generated in.
+#' @slot tree_number An \code{integer} representing the order in which the tree has been built in the model.
+#' @slot tree_class An \code{integer} representing tree's class, starting with 1.
+#' @slot root_node_id An \code{integer} representing number of the root node (may differ from 0).
+#' @aliases H2OTree
+#' @export
+setClass(
+  "H2OTree",
+  representation(
+    left_children = "integer",
+    right_children = "integer",
+    descriptions = "character",
+    model_name = "character",
```

Comment:
this should be consistent with other naming, use `model_id` instead

Reply:
Resolved:

```R
Slot "model_id":
[1] "gbm_trees_model"
```